pipeline-modules/app-service/azure/java-build.yaml

```diff
@@ -0,0 +1,25 @@
+provisioner: azure
+name: java-build
+version: 1
+revision: 1
+displayName: build jars with maven
```

Comment:
Shouldn't this name be more readable:
Build JARs with Maven?

Reply:
updated